### DISCUSSION TITLE
Does composition support Svelte?

### DISCUSSION THREAD

**Discussion by marcel0ll** [storybookjs/storybook#15157] [General] [answered]
Does composition support Svelte and the svelte-sfc?

I am trying out the new composition feature, but it seems that stories.json is empty for a project using svelte.js

**Comment by shilman** [comment, marked as answer]
Are you using `sb extract` or the experimental built in stories.json? The former should support everything; the latter has lots of limitations (which is why it's hidden behind a feature flag and experimental).

If you're having problems with `sb extract`, can you share a reproduction?

**Reply by marcel0ll** [reply]
I am using the feature flag. I will check `sb extract`

**Reply by shilman** [reply]
Are you using the svelte native story syntax? If so, that's not yet supported by the feature flag.
